include/wolfssl/wolfcrypt/settings.h

```diff
@@ -1797,25 +1797,25 @@ extern void uITRON4_free(void *p) ;
     #define STRING_USER
     #define XSTRCASECMP(s1,s2) strcasecmp((s1),(s2))
     #define XSTRCMP(s1,s2) strcmp((s1),(s2))
-    #define XSTRLEN(pstr) ((CPU_SIZE_T)Str_Len((CPU_CHAR *)(pstr)))
+    #define XSTRLEN(pstr) ((CPU_SIZE_T)Str_Len((CPunsigned char *)(pstr)))
```

Comment:
What does this data type mean?

Also, this settings.h header is one that you wrote, or is it copied verbatim from wolfssl?

Reply:
Well spotted as find and replace substituted U_CHAR as well as u_char! Fixed. Settings.h copied verbatim from WolfSSL

Reply:
Search-and-replace -- that explains it!

There are a handful more instances in the same settings.h that need to be reverted, I think. Please have a look.

Reply:
All done now!

Reply:
Cheers! Approved.